core/src/main/scala/org/ensime/core/PositionLocator.scala

```diff
@@ -0,0 +1,33 @@
+// Copyright: 2010 - 2016 https://github.com/ensime/ensime-server/graphs
+// License: http://www.gnu.org/licenses/gpl-3.0.en.html
+package org.ensime.core
+
+import scala.tools.nsc.interactive.Global
+
+trait PositionLocator {
```

Comment:
if this file has been derived from scala-ide it needs to be properly attributed to them